### PR TITLE
added HorizontalStoreCard to the library

### PR DESCRIPTION
**Pull request by sahandfarahani**
storereviewcard

### CODE REVIEW

File: src/Containers/PictureCard.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    height?: string;
+    borderradiustop?: string;
+    borderradiusbottom?: string;
+}
+
+const Card = styled(C)<CardProps>`
+    width: ${({ width }) => (width ? width : 'auto')};
+    height: ${({ height }) => (height ? height : 'auto')};
+    padding: 0px; //overriding styles
+    ${({ borderradiustop, borderradiusbottom }): string => `
```
Comment (fpunny) [inline src/Containers/PictureCard.tsx:79]: props are optional but they are being used as if they are required?

File: src/Containers/PictureCard.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+const ImageWrapper = styled.div`
+    position: relative;
+    ${flex('column')}
+`;
+
+const Image = styled.img<CardProps>`
+    width: 100%;
+    height: ${({ height }) => (height ? height : 'auto')};
+    ${({ borderradiustop, borderradiusbottom }): string => `
```
Comment (fpunny) [inline src/Containers/PictureCard.tsx:95]: Same as above

File: src/Containers/PictureCard.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    font-size: 0.75rem;
+    font-weight: bold;
+`;
+
+const Tag = styled.span`
+    ${flex('row', 'center')}
+    display: inline-flex;
+    background-color: white;
+    ${({ theme }): string => `
+    box-shadow: ${theme.depth[1]};
```
Comment (fpunny) [inline src/Containers/PictureCard.tsx:118]: indent

File: src/Containers/PictureCard.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    ${flex('row', 'flex-start', 'stretch')}
+    ${position('absolute', '0 0 10px 10px', 'auto', 'auto', 0, 0)}
+    ${({ theme }): string => `
+        color:${theme.colors.text};
+    `}
+    font-size: 0.75rem;
+    font-weight: bold;
+`;
+
+const Tag = styled.span`
```
Comment (fpunny) [inline src/Containers/PictureCard.tsx:113]: Can we not have and/or use a tag component in the ui kit?

File: src/Containers/PictureCard.tsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                        )}
+                    </Tags>
+                )}
+            </ImageWrapper>
+            {children && <Content>{children}</Content>}
+        </Card>
+    );
+};
+
+interface CardProps {
```
Comment (fpunny) [inline src/Containers/PictureCard.tsx:68]: If possible have PictureCard extend this to keep things DRY

File: src/Containers/PictureCard.tsx (hunk 2 above)
Comment (sahandfarahani) [reply src/Containers/PictureCard.tsx:95]: should I replace it with 
    border-top-left-radius:     ${({ borderradiustop }) => (borderradiustop ? borderradiustop : '0px')};
    border-top-right-radius:    ${({ borderradiustop }) => (borderradiustop ? borderradiustop : '0px')};
    border-bottom-left-radius:  ${({ borderradiusbottom }) => (borderradiusbottom ? borderradiusbottom : '0px')};
    border-bottom-right-radius: ${({ borderradiusbottom }) => (borderradiusbottom ? borderradiusbottom : '0px')};

?

File: src/Containers/PictureCard.tsx (hunk 4 above)
Comment (sahandfarahani) [reply src/Containers/PictureCard.tsx:113]: Good Idea

File: src/Containers/PictureCard.tsx (hunk 5 above)
Comment (sahandfarahani) [reply src/Containers/PictureCard.tsx:68]: you mean
interface PictureCardProps
    extends MainInterface,
        ResponsiveInterface,
        CardProps { etc.

and then 
export const PictureCard: React.FC<PictureCardProps> = ({etc.

right?

File: src/Containers/PictureCard.tsx (hunk 2 above)
Comment (fpunny) [reply src/Containers/PictureCard.tsx]: Use default object props

File: src/Containers/PictureCard.tsx (hunk 5 above)
Comment (fpunny) [reply src/Containers/PictureCard.tsx:72]: yes

**Comment by ralph-dev** [comment]
@sahandfarahani just from a design side, make the circles between the grey text tags in the middle vertically (Instead of top alinged)

**Pull request merged**